framework/src/mesh/MooseMesh.C

```diff
@@ -724,8 +814,12 @@ ConstNodeRange *
 MooseMesh::getLocalNodeRange()
 {
   if (!_local_node_range)
+  {
+    TIME_SECTION(_get_local_node_range_timer);
```

Comment:
Should the scoped timer always be used? For really short timers like this, it might be faster to use push/pop when we know it won't throw. Thoughts?